lib/handlers/api.js

```diff
@@ -31,7 +31,7 @@ import { logger } from '../logger';
 import { getShortenerTypeByKey } from '../shortener';
 import * as utils from '../utils';
 
-import { AsmDocsHandler } from './asm-docs-api';
+import { AsmDocsHandler } from './asm-docs-api-arm32';
```

Comment:
This is what you mentioned before when you (currently) hardcode to arm32? :+1:

We can definitely help with that :)

Essentially we can include both here, and then "mount" them at line 63. But we can help/do this.